src/tests/plans/D3MAaveV2TypeRateTargetPlan.t.sol

```diff
@@ -47,7 +47,7 @@ contract D3MAaveV2TypeRateTargetPlanWrapper is D3MAaveV2TypeRateTargetPlan {
     }
 }
 
-contract D3MAaveV2TypeRateTargetPlanTest is D3MPlanBaseTest {
+abstract contract D3MAaveV2TypeRateTargetPlanTest is D3MPlanBaseTest {
```

Comment:
Is this intended to make these tests not run at all?

Reply:
Yes, it was broken and we don't use Aave V2 anymore so I didn't see any value in fixing it.

Reply:
Can we add a comment that this is the intention? wasn't clear to me at first why it was made abstract.